### PR TITLE
add noopener and fix EnsAddress fallback

### PR DESCRIPTION
**Pull request by MixailE**

### CODE REVIEW

File: src/components/navbar/Wallet.tsx [hunk 1]
```diff
@@ -73,12 +73,17 @@ export default function () {
...
-            window
-              .open(getEtherscanAddressUrl(account, Network.Goerli), '_blank')
-              ?.focus()
-          } else {
-            await WalletStore.connect(true)
+            const newWindow = window.open(
+              getEtherscanAddressUrl(account, Network.Goerli),
+              '_blank'
+            )
+            if (newWindow) {
```
Comment (T-Damer) [inline src/components/navbar/Wallet.tsx:80]: Maybe `if (!newWindow) return`

File: src/components/EnsAddress.tsx [hunk 2]
```diff
@@ -13,27 +13,33 @@ interface EnsAddressProps {
 function EnsAddressSuspended({
   address,
   truncate,
-}: EnsAddressProps & { truncate?: boolean }) {
+  truncateSize,
```
Comment (T-Damer) [inline src/components/EnsAddress.tsx:16]: If you're using `truncateSize`, this means that you want to truncate, so you don't need two props, I guess

**Review by T-Damer — APPROVED** [review]

File: src/components/EnsAddress.tsx (hunk 2 above)
Comment (smyzin) [reply src/components/EnsAddress.tsx:16]: Agreed, otherwise looks good

Comment (MixailE) [reply src/components/EnsAddress.tsx:16]: This is not true, `truncate` is needed for `ensName` and only when the screen size is large :)

**Review by smyzin — APPROVED** [review]
Lgtm

**Pull request merged**